src/components/CCard.vue

```diff
@@ -40,15 +40,17 @@
     </div>
 
     <v-divider
-      v-if="scrollable && (title || subtitle || slots['header'])"
+      v-if="scrollingEnabled && (title || subtitle || slots['header'])"
       class="mb-0"
     ></v-divider>
 
     <v-card-text
       ref="cardTextComponent"
       class="ma-0 px-0"
-      :class="scrollingContentVerticalPadding && scrollable ? 'py-4' : 'py-0'"
-      style="overflow-y: auto"
+      :class="
+        scrollingContentVerticalPadding && scrollingEnabled ? 'py-4' : 'py-0'
+      "
+      :style="scrollingEnabled ? 'overflow-y: scroll' : 'overflow-y: auto'"
```

Comment:
I believe this should be `scrollable ? 'overflow-y: scroll'...` technically, shouldn't it?

Reply:
sounds reasonable